packages/postcss/mixin.core.js

```diff
@@ -141,24 +156,36 @@ class PostCSSMixin extends Mixin {
       oneOf: [
         {
           resourceQuery: /global/,
-          use: {
-            loader: require.resolve('css-loader'),
-            options: {
-              ...cssLoaderGlobalOptions,
-              importLoaders: 0,
-              onlyLocals: true,
+          use: [
+            {
+              loader: require.resolve('css-loader'),
+              options: {
+                ...cssLoaderGlobalOptions,
+                modules: {
+                  auto: false,
+                  exportOnlyLocals: true,
+                },
+                importLoaders: 1,
+              },
             },
-          },
+            getPostCssLoader(this.config.browsers),
```

Comment:
As mentioned above, `postcss-loader` is applied here too, to get the imports straight.